unicodetools/data/ucd/dev/VerticalOrientation.txt

```diff
@@ -1499,7 +1499,9 @@ D7B0..D7C6     ; U  # Lo    [23] HANGUL JUNGSEONG O-YEO..HANGUL JUNGSEONG ARAEA-
 D7C7..D7CA     ; U  # Cn     [4] <reserved-D7C7>..<reserved-D7CA>
 D7CB..D7FB     ; U  # Lo    [49] HANGUL JONGSEONG NIEUN-RIEUL..HANGUL JONGSEONG PHIEUPH-THIEUTH
 D7FC..D7FF     ; U  # Cn     [4] <reserved-D7FC>..<reserved-D7FF>
-D800..DFFF     ; R  # Cs  [2048] <surrogate-D800>..<surrogate-DFFF>
+D800..DB7F     ; R  # Cs   [896] <surrogate-D800>..<surrogate-DB7F>
```

Comment:
This seems like an odd change. Harmless, but I'm curious as to the cause.

Reply:
Ken merged two tools into one for consistent formatting of EAW.txt, LB.txt, and VO.txt.
That tool is not (yet) tracked in this repo, so you don't see a corresponding code change here.

Reply:
This is actually a formal block boundary. The code is consistent in honoring block boundaries for splitting ranges, as that provides clarity when a range of values crosses an arbitrary block boundary. This business with the surrogate blocks is just a matter of consistency then.